cypress/e2e/loginSuccess.cy.js

```diff
@@ -0,0 +1,9 @@
+describe("loginSuccess", () => {
+  it("can log in with the login form with valid credentials", () => {
+    cy.visit("/").wait(300);
+    cy.get("#registerModal button").contains("Login").wait(300).click().wait(500);
+    cy.get(`input#loginEmail[name="email"]`).type("cypresstest@noroff.no");
+    cy.get(`input#loginPassword[name="password"]`).type("cypresstest");
+    cy.get(`button[type="submit"]`).contains("Login").click();
```

Comment:
There should really be a step after this, like checking storage for the token and expecting null